File: Task Tracker C#/contest/contestDomain/dto/TaskDTO.cs
using contestDomain.enums;
using Type = contestDomain.enums.Type;

namespace contestDomain.dto
{
    [Serializable]
    public class TaskDTO
    {
        public int taskID { get; set; }
        public Type type { get; set; }
        public AgeGroup ageGroup { get; set; }
        public int Enrolled { get; set; }

        public TaskDTO(Type type, AgeGroup ageGroup, int enrolled)
        {
            this.type = type;
            this.ageGroup = ageGroup;
            this.Enrolled = enrolled;
        }

        public TaskDTO(int id, Type type, AgeGroup ageGroup, int enrolled)
        {
            this.taskID = id;
            this.type = type;
            this.ageGroup = ageGroup;
            this.Enrolled = enrolled;
        }
    }
}

File: Task Tracker Java/contest_java/contestDomain/src/main/java/com/example/dto/TaskDTO.java
package com.example.dto;
import com.example.enums.Type;
import com.example.enums.AgeGroup;

import java.io.Serializable;
import java.util.Objects;

public class TaskDTO implements Serializable {
    private int TaskID;
    private Type Type;
    private AgeGroup AgeGroup;
    private int Enrolled;

    public TaskDTO(int taskID, Type type, AgeGroup ageGroup, int enrolled) {
        TaskID = taskID;
        Type = type;
        AgeGroup = ageGroup;
        Enrolled = enrolled;
    }

    public int getTaskID() {
        return TaskID;
    }

    public Type getType() {
        return Type;
    }

    public AgeGroup getAgeGroup() {
        return AgeGroup;
    }

    public int getEnrolled() {
        return Enrolled;
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        TaskDTO taskDTO = (TaskDTO) o;
        return TaskID == taskDTO.TaskID && Enrolled == taskDTO.Enrolled && Type == taskDTO.Type && AgeGroup == taskDTO.AgeGroup;
    }

    @Override
    public int hashCode() {
        return Objects.hash(TaskID, Type, AgeGroup, Enrolled);
    }
}
